diff -Nru psi-plus-0.15~svn3753/debian/control psi-plus-0.15~svn3775/debian/control
--- psi-plus-0.15~svn3753/debian/control	2010-12-23 11:58:01.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/control	2011-03-29 15:05:08.000000000 +0400
@@ -11,9 +11,7 @@
 
 Package: psi-plus
 Architecture: any
-Conflicts: psi
 Depends: psi-plus-common, ${shlibs:Depends}, ${misc:Depends}
-Replaces: psi
 Description: Qt-based jabber client (basic version)
  This package contains basic version of Psi+.
  .
@@ -47,9 +45,7 @@
 
 Package: psi-plus-webkit
 Architecture: any
-Conflicts: psi
 Depends: psi-plus-common, ${shlibs:Depends}, ${misc:Depends}
-Replaces: psi
 Description: Qt-based jabber client (WebKit version)
  This package contains WebKit version of Psi+.
  .
@@ -84,7 +80,6 @@
 Package: psi-plus-common
 Architecture: all
 Recommends: sox, libqca2-plugin-ossl, psi-plus-plugins, psi-plus-icons, psi-plus-skins
-Conflicts: psi-translations
 Breaks: psi-plus (<< 0.15~svn3057)
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Suggests: libqca2-plugin-gnupg, xdg-utils
diff -Nru psi-plus-0.15~svn3753/debian/psi.1 psi-plus-0.15~svn3775/debian/psi.1
--- psi-plus-0.15~svn3753/debian/psi.1	2010-04-25 18:57:33.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi.1	1970-01-01 03:00:00.000000000 +0300
@@ -1,21 +0,0 @@
-.TH PSI 1 "November  1, 2001"
-.\" Please adjust this date whenever revising the manpage.
-.SH NAME
-psi \- jabber client
-.SH SYNOPSIS
-.B psi
-.SH DESCRIPTION
-.B psi 
-is a free instant messaging application designed for the Jabber
-(XMPP) IM network (including Google Talk). It has full Unicode support, 
-and supports most important XMPP features like file transfer, group chat,
-encryption etc.
-.PP
-Psi relies on the qca2 library for its encryption features. Therefore,
-it's necessary to install libqca2-plugin-ossl for SSL (client-to-server)
-encryption and libqca2-plugin-gnupg for PGP (end-to-end) encryption.
-.PP
-A tutorial can be found on the web at http://psi-im.org/wiki/Tutorial
-.SH AUTHOR
-This manual page was written by Jan Niehusmann <jan@debian.org>,
-for the Debian GNU/Linux system (but may be used by others).
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus.1 psi-plus-0.15~svn3775/debian/psi-plus.1
--- psi-plus-0.15~svn3753/debian/psi-plus.1	1970-01-01 03:00:00.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus.1	2010-04-25 18:57:33.000000000 +0400
@@ -0,0 +1,21 @@
+.TH PSI 1 "November  1, 2001"
+.\" Please adjust this date whenever revising the manpage.
+.SH NAME
+psi \- jabber client
+.SH SYNOPSIS
+.B psi
+.SH DESCRIPTION
+.B psi 
+is a free instant messaging application designed for the Jabber
+(XMPP) IM network (including Google Talk). It has full Unicode support, 
+and supports most important XMPP features like file transfer, group chat,
+encryption etc.
+.PP
+Psi relies on the qca2 library for its encryption features. Therefore,
+it's necessary to install libqca2-plugin-ossl for SSL (client-to-server)
+encryption and libqca2-plugin-gnupg for PGP (end-to-end) encryption.
+.PP
+A tutorial can be found on the web at http://psi-im.org/wiki/Tutorial
+.SH AUTHOR
+This manual page was written by Jan Niehusmann <jan@debian.org>,
+for the Debian GNU/Linux system (but may be used by others).
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-common.install psi-plus-0.15~svn3775/debian/psi-plus-common.install
--- psi-plus-0.15~svn3753/debian/psi-plus-common.install	2010-10-02 20:43:57.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi-plus-common.install	2011-03-29 13:28:59.000000000 +0400
@@ -1 +1 @@
-lang/ru/psi_ru.qm /usr/share/psi
+lang/ru/psi_ru.qm /usr/share/psi-plus
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-content-download.install psi-plus-0.15~svn3775/debian/psi-plus-content-download.install
--- psi-plus-0.15~svn3753/debian/psi-plus-content-download.install	2010-09-29 01:16:59.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi-plus-content-download.install	2011-03-29 13:28:54.000000000 +0400
@@ -1 +1 @@
-src/plugins/generic/contentdownloaderplugin/libcontentdownloaderplugin.so usr/lib/psi/plugins
+src/plugins/generic/contentdownloaderplugin/libcontentdownloaderplugin.so usr/lib/psi-plus/plugins
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus.install psi-plus-0.15~svn3775/debian/psi-plus.install
--- psi-plus-0.15~svn3753/debian/psi-plus.install	2010-10-02 20:43:46.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi-plus.install	2011-03-29 13:28:50.000000000 +0400
@@ -1 +1 @@
-debian/psi.xpm /usr/share/psi/
+debian/psi-plus.xpm /usr/share/psi-plus/
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus.manpages psi-plus-0.15~svn3775/debian/psi-plus.manpages
--- psi-plus-0.15~svn3753/debian/psi-plus.manpages	2010-04-25 18:57:33.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi-plus.manpages	2011-03-29 12:33:35.000000000 +0400
@@ -1 +1 @@
-debian/psi.1
+debian/psi-plus.1
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus.menu psi-plus-0.15~svn3775/debian/psi-plus.menu
--- psi-plus-0.15~svn3753/debian/psi-plus.menu	2010-04-25 16:59:00.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi-plus.menu	2011-03-29 12:33:51.000000000 +0400
@@ -1,3 +1,3 @@
 ?package(psi-plus):needs="X11" section="Applications/Network/Communication"\
-  title="Psi+" command="/usr/bin/psi"\
-  icon="/usr/share/psi/psi.xpm"
+  title="Psi+" command="/usr/bin/psi-plus"\
+  icon="/usr/share/psi-plus/psi-plus.xpm"
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-plugins.dirs psi-plus-0.15~svn3775/debian/psi-plus-plugins.dirs
--- psi-plus-0.15~svn3753/debian/psi-plus-plugins.dirs	2010-04-25 18:57:33.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi-plus-plugins.dirs	2011-03-29 13:29:05.000000000 +0400
@@ -1 +1 @@
-usr/lib/psi/plugins
+usr/lib/psi-plus/plugins
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-plugins.install psi-plus-0.15~svn3775/debian/psi-plus-plugins.install
--- psi-plus-0.15~svn3753/debian/psi-plus-plugins.install	2010-04-25 18:57:33.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi-plus-plugins.install	2011-03-29 13:28:44.000000000 +0400
@@ -1 +1 @@
-src/plugins/generic/*/*.so usr/lib/psi/plugins
+src/plugins/generic/*/*.so usr/lib/psi-plus/plugins
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-skins.install psi-plus-0.15~svn3775/debian/psi-plus-skins.install
--- psi-plus-0.15~svn3753/debian/psi-plus-skins.install	2010-12-13 23:10:59.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus-skins.install	2011-03-29 13:28:40.000000000 +0400
@@ -1 +1 @@
-skins usr/share/psi
+skins usr/share/psi-plus
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-webkit.1 psi-plus-0.15~svn3775/debian/psi-plus-webkit.1
--- psi-plus-0.15~svn3753/debian/psi-plus-webkit.1	1970-01-01 03:00:00.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus-webkit.1	2010-11-16 23:50:33.000000000 +0300
@@ -0,0 +1,23 @@
+.TH PSI 1 "November  1, 2001"
+.\" Please adjust this date whenever revising the manpage.
+.SH NAME
+psi \- jabber client
+.SH SYNOPSIS
+.B psi
+.SH DESCRIPTION
+.B psi 
+is a free instant messaging application designed for the Jabber
+(XMPP) IM network (including Google Talk). It has full Unicode support, 
+and supports most important XMPP features like file transfer, group chat,
+encryption etc.
+.PP
+Psi relies on the qca2 library for its encryption features. Therefore,
+it's necessary to install libqca2-plugin-ossl for SSL (client-to-server)
+encryption and libqca2-plugin-gnupg for PGP (end-to-end) encryption.
+.PP
+A tutorial can be found on the web at http://psi-im.org/wiki/Tutorial
+.PP
+This version build whis webkit
+.SH AUTHOR
+This manual page was written by Jan Niehusmann <jan@debian.org>,
+for the Debian GNU/Linux system (but may be used by others).
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-webkit.desktop psi-plus-0.15~svn3775/debian/psi-plus-webkit.desktop
--- psi-plus-0.15~svn3753/debian/psi-plus-webkit.desktop	1970-01-01 03:00:00.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus-webkit.desktop	2011-03-29 14:53:03.000000000 +0400
@@ -0,0 +1,55 @@
+[Desktop Entry]
+# This is the spec version, *not* the application version
+Version=1.0
+Type=Application
+Name=Psi Plus Webkit
+GenericName=Jabber Client
+Comment=Communicate over the Jabber network
+Icon=psi-plus
+Exec=psi-plus-webkit
+Terminal=false
+Categories=Network;InstantMessaging;Qt;
+
+# Translations
+GenericName[be]=Кліент Jabber
+Comment[be]=Размаўляйце ў сетцы Jabber
+GenericName[bg]=Jabber Клиент
+Comment[bg]=Комуникирайте през Jabber мрежата
+GenericName[ca]=Client Jabber
+Comment[ca]=Comuniqui's a través de la xarxa Jabber
+GenericName[cs]=Jabber Klient
+Comment[cs]=Komunikujte přes Jabber po síti
+GenericName[de]=Jabber-Programm
+Comment[de]=Im Jabber-Netzwerk kommunizieren
+GenericName[eo]=Ĵabbera Klientilo
+Comment[eo]=Komunikiĝu per la ĵabbera reto
+GenericName[es]=Cliente Jabber
+Comment[es]=Comunícate a través de la red Jabber
+GenericName[fr]=Client Jabber
+Comment[fr]=Communiquez via le réseau Jabber
+GenericName[it]=Client Jabber
+Comment[it]=Comunica sulla rete Jabber
+GenericName[ja]=Jabberクライアント
+Comment[ja]=Jabberネットーワーク通信
+GenericName[mk]=Клиент за Jabber
+Comment[mk]=Комуницирајте преку Jabber мрежата
+GenericName[nl]=Jabber Client
+Comment[nl]=Communiceer over het Jabber netwerk
+GenericName[pl]=Klient XMPP
+Comment[pl]=Komunikacja poprzez sieć XMPP
+GenericName[pt_BR]=Cliente Jabber
+Comment[pt_BR]=Comunique-se através da rede Jabber
+GenericName[ru]=Jabber-клиент
+Comment[ru]=Общение в сети Jabber
+GenericName[sl]=Odjemalec za Jabber/XMPP
+Comment[sl]=Takojšnje sporočanje prek omrežij Jabber/XMPP
+GenericName[sv]=Jabberklient
+Comment[sv]=Kommunicera över Jabbernätverket
+GenericName[uk]=Jabber клієнт
+Comment[uk]=Програма для спілкування в мережі jabber.
+GenericName[ur_PK]=جیبر وصول کار
+Comment[ur_PK]=جیبر نیٹ ورک پر مواصلت کریں
+GenericName[vi]=Ứng dụng khách Jabber
+Comment[vi]=Liên lạc qua mạng Jabber nhé
+GenericName[zh_TW]=Jabber 客戶端
+Comment[zh_TW]=在 Jabber 網路上交談
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-webkit.install psi-plus-0.15~svn3775/debian/psi-plus-webkit.install
--- psi-plus-0.15~svn3753/debian/psi-plus-webkit.install	2010-11-16 23:47:47.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus-webkit.install	2011-03-29 13:28:36.000000000 +0400
@@ -1 +1 @@
-debian/psi-webkit.xpm /usr/share/psi/
+debian/psi-plus-webkit.xpm /usr/share/psi-plus/
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-webkit.manpages psi-plus-0.15~svn3775/debian/psi-plus-webkit.manpages
--- psi-plus-0.15~svn3753/debian/psi-plus-webkit.manpages	2010-11-16 23:49:49.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus-webkit.manpages	2011-03-29 12:33:39.000000000 +0400
@@ -1 +1 @@
-debian/psi-webkit.1
+debian/psi-plus-webkit.1
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-webkit.menu psi-plus-0.15~svn3775/debian/psi-plus-webkit.menu
--- psi-plus-0.15~svn3753/debian/psi-plus-webkit.menu	2010-11-16 23:48:28.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus-webkit.menu	2011-03-29 14:53:29.000000000 +0400
@@ -1,3 +1,3 @@
 ?package(psi-plus-webkit):needs="X11" section="Applications/Network/Communication"\
-  title="Psi+" command="/usr/bin/psi-webkit"\
-  icon="/usr/share/psi/psi-webkit.xpm"
+  title="Psi+" command="/usr/bin/psi-plus-webkit"\
+  icon="/usr/share/psi-plus/psi-plus-webkit.xpm"
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus-webkit.xpm psi-plus-0.15~svn3775/debian/psi-plus-webkit.xpm
--- psi-plus-0.15~svn3753/debian/psi-plus-webkit.xpm	1970-01-01 03:00:00.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus-webkit.xpm	2010-11-16 13:54:57.000000000 +0300
@@ -0,0 +1,47 @@
+/* XPM */
+static char *noname[] = {
+/* width height ncolors chars_per_pixel */
+"32 32 8 1",
+/* colors */
+"  c #000000",
+". c #CA8AC9",
+"X c #57ACDA",
+"o c #ABDBF4",
+"O c #72BDE6",
+"@ c #87CCF2",
+"# c #CDE9F8",
+"+ c None",
+/* pixels */
+"+        ++++      +++         +",
+"          ++        +           ",
+"  ######   +  ####      #####X  ",
+"  #oooooX     #ooX     #ooooXX  ",
+"    #o@@OX    #oOX    #o@@OX    ",
+"+    #o@OX    #oOX    #o@OX    +",
+"+++  #o@OX    #oOX    #o@OX  +++",
+"+++  #o@OX    #oOX    #o@OX  +++",
+"+++  #o@@OX   #oOX   #o@@OX  +++",
+"+++  #o@@OX   #oOX   #o@@OX  +++",
+"+++   #o@OX   #oOX   #o@OX   +++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX    ++",
+"++++   #o@OX  #oOX  #o@OX      +",
+"+++++  #o@OX  #oOX  #o@OX   ..  ",
+"+++++   Xo@OX #oOX #o@OX ..  .. ",
+"++++++   Xo@@@@@OOOOOO..  .. .. ",
+"+++++++   XXXooo@OOXXX .. .. .. ",
+"++++++++     XXoOXX    .. .. .. ",
+"+++++++++     #oOX     .. .. .. ",
+"++++++++++++  #oOX  + ..  .. .. ",
+"++++++++++++  #oOX  +    ..  .. ",
+"++++++++++++  #oOX  ++      ..  ",
+"++++++++++++  #oOX  ++++       +",
+"++++++++++++  #oOX  ++++++    ++",
+"++++++++++++  #oOX  ++++++++++++",
+"++++++++++++  #XXX  ++++++++++++",
+"++++++++++++        ++++++++++++",
+"+++++++++++++      +++++++++++++"
+};
diff -Nru psi-plus-0.15~svn3753/debian/psi-plus.xpm psi-plus-0.15~svn3775/debian/psi-plus.xpm
--- psi-plus-0.15~svn3753/debian/psi-plus.xpm	1970-01-01 03:00:00.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-plus.xpm	2010-04-25 18:57:33.000000000 +0400
@@ -0,0 +1,47 @@
+/* XPM */
+static char *noname[] = {
+/* width height ncolors chars_per_pixel */
+"32 32 8 1",
+/* colors */
+"  c #000000",
+". c #CA8AC9",
+"X c #57ACDA",
+"o c #ABDBF4",
+"O c #72BDE6",
+"@ c #87CCF2",
+"# c #CDE9F8",
+"+ c None",
+/* pixels */
+"+        ++++      +++         +",
+"          ++        +           ",
+"  ######   +  ####      #####X  ",
+"  #oooooX     #ooX     #ooooXX  ",
+"    #o@@OX    #oOX    #o@@OX    ",
+"+    #o@OX    #oOX    #o@OX    +",
+"+++  #o@OX    #oOX    #o@OX  +++",
+"+++  #o@OX    #oOX    #o@OX  +++",
+"+++  #o@@OX   #oOX   #o@@OX  +++",
+"+++  #o@@OX   #oOX   #o@@OX  +++",
+"+++   #o@OX   #oOX   #o@OX   +++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX  ++++",
+"++++  #o@OX   #oOX   #o@OX    ++",
+"++++   #o@OX  #oOX  #o@OX      +",
+"+++++  #o@OX  #oOX  #o@OX   ..  ",
+"+++++   Xo@OX #oOX #o@OX ..  .. ",
+"++++++   Xo@@@@@OOOOOO..  .. .. ",
+"+++++++   XXXooo@OOXXX .. .. .. ",
+"++++++++     XXoOXX    .. .. .. ",
+"+++++++++     #oOX     .. .. .. ",
+"++++++++++++  #oOX  + ..  .. .. ",
+"++++++++++++  #oOX  +    ..  .. ",
+"++++++++++++  #oOX  ++      ..  ",
+"++++++++++++  #oOX  ++++       +",
+"++++++++++++  #oOX  ++++++    ++",
+"++++++++++++  #oOX  ++++++++++++",
+"++++++++++++  #XXX  ++++++++++++",
+"++++++++++++        ++++++++++++",
+"+++++++++++++      +++++++++++++"
+};
diff -Nru psi-plus-0.15~svn3753/debian/psi-webkit.1 psi-plus-0.15~svn3775/debian/psi-webkit.1
--- psi-plus-0.15~svn3753/debian/psi-webkit.1	2010-11-16 23:50:33.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-webkit.1	1970-01-01 03:00:00.000000000 +0300
@@ -1,23 +0,0 @@
-.TH PSI 1 "November  1, 2001"
-.\" Please adjust this date whenever revising the manpage.
-.SH NAME
-psi \- jabber client
-.SH SYNOPSIS
-.B psi
-.SH DESCRIPTION
-.B psi 
-is a free instant messaging application designed for the Jabber
-(XMPP) IM network (including Google Talk). It has full Unicode support, 
-and supports most important XMPP features like file transfer, group chat,
-encryption etc.
-.PP
-Psi relies on the qca2 library for its encryption features. Therefore,
-it's necessary to install libqca2-plugin-ossl for SSL (client-to-server)
-encryption and libqca2-plugin-gnupg for PGP (end-to-end) encryption.
-.PP
-A tutorial can be found on the web at http://psi-im.org/wiki/Tutorial
-.PP
-This version build whis webkit
-.SH AUTHOR
-This manual page was written by Jan Niehusmann <jan@debian.org>,
-for the Debian GNU/Linux system (but may be used by others).
diff -Nru psi-plus-0.15~svn3753/debian/psi-webkit.desktop psi-plus-0.15~svn3775/debian/psi-webkit.desktop
--- psi-plus-0.15~svn3753/debian/psi-webkit.desktop	2010-11-11 00:03:44.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-webkit.desktop	1970-01-01 03:00:00.000000000 +0300
@@ -1,55 +0,0 @@
-[Desktop Entry]
-# This is the spec version, *not* the application version
-Version=1.0
-Type=Application
-Name=Psi Plus Webkit
-GenericName=Jabber Client
-Comment=Communicate over the Jabber network
-Icon=psi-webkit
-Exec=psi-webkit
-Terminal=false
-Categories=Network;InstantMessaging;Qt;
-
-# Translations
-GenericName[be]=Кліент Jabber
-Comment[be]=Размаўляйце ў сетцы Jabber
-GenericName[bg]=Jabber Клиент
-Comment[bg]=Комуникирайте през Jabber мрежата
-GenericName[ca]=Client Jabber
-Comment[ca]=Comuniqui's a través de la xarxa Jabber
-GenericName[cs]=Jabber Klient
-Comment[cs]=Komunikujte přes Jabber po síti
-GenericName[de]=Jabber-Programm
-Comment[de]=Im Jabber-Netzwerk kommunizieren
-GenericName[eo]=Ĵabbera Klientilo
-Comment[eo]=Komunikiĝu per la ĵabbera reto
-GenericName[es]=Cliente Jabber
-Comment[es]=Comunícate a través de la red Jabber
-GenericName[fr]=Client Jabber
-Comment[fr]=Communiquez via le réseau Jabber
-GenericName[it]=Client Jabber
-Comment[it]=Comunica sulla rete Jabber
-GenericName[ja]=Jabberクライアント
-Comment[ja]=Jabberネットーワーク通信
-GenericName[mk]=Клиент за Jabber
-Comment[mk]=Комуницирајте преку Jabber мрежата
-GenericName[nl]=Jabber Client
-Comment[nl]=Communiceer over het Jabber netwerk
-GenericName[pl]=Klient XMPP
-Comment[pl]=Komunikacja poprzez sieć XMPP
-GenericName[pt_BR]=Cliente Jabber
-Comment[pt_BR]=Comunique-se através da rede Jabber
-GenericName[ru]=Jabber-клиент
-Comment[ru]=Общение в сети Jabber
-GenericName[sl]=Odjemalec za Jabber/XMPP
-Comment[sl]=Takojšnje sporočanje prek omrežij Jabber/XMPP
-GenericName[sv]=Jabberklient
-Comment[sv]=Kommunicera över Jabbernätverket
-GenericName[uk]=Jabber клієнт
-Comment[uk]=Програма для спілкування в мережі jabber.
-GenericName[ur_PK]=جیبر وصول کار
-Comment[ur_PK]=جیبر نیٹ ورک پر مواصلت کریں
-GenericName[vi]=Ứng dụng khách Jabber
-Comment[vi]=Liên lạc qua mạng Jabber nhé
-GenericName[zh_TW]=Jabber 客戶端
-Comment[zh_TW]=在 Jabber 網路上交談
diff -Nru psi-plus-0.15~svn3753/debian/psi-webkit.xpm psi-plus-0.15~svn3775/debian/psi-webkit.xpm
--- psi-plus-0.15~svn3753/debian/psi-webkit.xpm	2010-11-16 13:54:57.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/psi-webkit.xpm	1970-01-01 03:00:00.000000000 +0300
@@ -1,47 +0,0 @@
-/* XPM */
-static char *noname[] = {
-/* width height ncolors chars_per_pixel */
-"32 32 8 1",
-/* colors */
-"  c #000000",
-". c #CA8AC9",
-"X c #57ACDA",
-"o c #ABDBF4",
-"O c #72BDE6",
-"@ c #87CCF2",
-"# c #CDE9F8",
-"+ c None",
-/* pixels */
-"+        ++++      +++         +",
-"          ++        +           ",
-"  ######   +  ####      #####X  ",
-"  #oooooX     #ooX     #ooooXX  ",
-"    #o@@OX    #oOX    #o@@OX    ",
-"+    #o@OX    #oOX    #o@OX    +",
-"+++  #o@OX    #oOX    #o@OX  +++",
-"+++  #o@OX    #oOX    #o@OX  +++",
-"+++  #o@@OX   #oOX   #o@@OX  +++",
-"+++  #o@@OX   #oOX   #o@@OX  +++",
-"+++   #o@OX   #oOX   #o@OX   +++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX    ++",
-"++++   #o@OX  #oOX  #o@OX      +",
-"+++++  #o@OX  #oOX  #o@OX   ..  ",
-"+++++   Xo@OX #oOX #o@OX ..  .. ",
-"++++++   Xo@@@@@OOOOOO..  .. .. ",
-"+++++++   XXXooo@OOXXX .. .. .. ",
-"++++++++     XXoOXX    .. .. .. ",
-"+++++++++     #oOX     .. .. .. ",
-"++++++++++++  #oOX  + ..  .. .. ",
-"++++++++++++  #oOX  +    ..  .. ",
-"++++++++++++  #oOX  ++      ..  ",
-"++++++++++++  #oOX  ++++       +",
-"++++++++++++  #oOX  ++++++    ++",
-"++++++++++++  #oOX  ++++++++++++",
-"++++++++++++  #XXX  ++++++++++++",
-"++++++++++++        ++++++++++++",
-"+++++++++++++      +++++++++++++"
-};
diff -Nru psi-plus-0.15~svn3753/debian/psi.xpm psi-plus-0.15~svn3775/debian/psi.xpm
--- psi-plus-0.15~svn3753/debian/psi.xpm	2010-04-25 18:57:33.000000000 +0400
+++ psi-plus-0.15~svn3775/debian/psi.xpm	1970-01-01 03:00:00.000000000 +0300
@@ -1,47 +0,0 @@
-/* XPM */
-static char *noname[] = {
-/* width height ncolors chars_per_pixel */
-"32 32 8 1",
-/* colors */
-"  c #000000",
-". c #CA8AC9",
-"X c #57ACDA",
-"o c #ABDBF4",
-"O c #72BDE6",
-"@ c #87CCF2",
-"# c #CDE9F8",
-"+ c None",
-/* pixels */
-"+        ++++      +++         +",
-"          ++        +           ",
-"  ######   +  ####      #####X  ",
-"  #oooooX     #ooX     #ooooXX  ",
-"    #o@@OX    #oOX    #o@@OX    ",
-"+    #o@OX    #oOX    #o@OX    +",
-"+++  #o@OX    #oOX    #o@OX  +++",
-"+++  #o@OX    #oOX    #o@OX  +++",
-"+++  #o@@OX   #oOX   #o@@OX  +++",
-"+++  #o@@OX   #oOX   #o@@OX  +++",
-"+++   #o@OX   #oOX   #o@OX   +++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX  ++++",
-"++++  #o@OX   #oOX   #o@OX    ++",
-"++++   #o@OX  #oOX  #o@OX      +",
-"+++++  #o@OX  #oOX  #o@OX   ..  ",
-"+++++   Xo@OX #oOX #o@OX ..  .. ",
-"++++++   Xo@@@@@OOOOOO..  .. .. ",
-"+++++++   XXXooo@OOXXX .. .. .. ",
-"++++++++     XXoOXX    .. .. .. ",
-"+++++++++     #oOX     .. .. .. ",
-"++++++++++++  #oOX  + ..  .. .. ",
-"++++++++++++  #oOX  +    ..  .. ",
-"++++++++++++  #oOX  ++      ..  ",
-"++++++++++++  #oOX  ++++       +",
-"++++++++++++  #oOX  ++++++    ++",
-"++++++++++++  #oOX  ++++++++++++",
-"++++++++++++  #XXX  ++++++++++++",
-"++++++++++++        ++++++++++++",
-"+++++++++++++      +++++++++++++"
-};
diff -Nru psi-plus-0.15~svn3753/debian/README.source psi-plus-0.15~svn3775/debian/README.source
--- psi-plus-0.15~svn3753/debian/README.source	2011-01-06 13:46:34.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/README.source	2011-03-29 15:04:07.000000000 +0400
@@ -6,3 +6,4 @@
 For Debian users:
 build scripts not in package, get last version from Ubuntu PPA
 https://launchpad.net/~psi-plus/+archive/ppa
+
diff -Nru psi-plus-0.15~svn3753/debian/rules psi-plus-0.15~svn3775/debian/rules
--- psi-plus-0.15~svn3753/debian/rules	2011-03-05 11:08:00.000000000 +0300
+++ psi-plus-0.15~svn3775/debian/rules	2011-03-29 13:28:08.000000000 +0400
@@ -27,22 +27,22 @@
 
 override_dh_auto_install:
 	$(MAKE) INSTALL_ROOT=$(CURDIR)/debian/psi-plus install
-	rm -f $(CURDIR)/debian/psi-plus/usr/bin/psi.debug #remove static debug file
+	rm -f $(CURDIR)/debian/psi-plus/usr/bin/psi-plus.debug #remove static debug file
 	#Making the same, but using webkit
 	$(MAKE) clean
 	./configure --prefix=/usr $(CONF) --enable-webkit
 	$(MAKE) $(MAKEOPT)
 	$(MAKE) INSTALL_ROOT=$(CURDIR)/debian/psi-plus-webkit install
-	rm $(CURDIR)/debian/psi-plus-webkit/usr/bin/psi
-	rm $(CURDIR)/debian/psi-plus-webkit/usr/share/applications/psi.desktop
-	cp $(CURDIR)/src/psi $(CURDIR)/debian/psi-plus-webkit/usr/bin/psi-webkit
-	cp $(CURDIR)/debian/psi-webkit.desktop $(CURDIR)/debian/psi-plus-webkit/usr/share/applications/
-	rm -f $(CURDIR)/debian/psi-plus-webkit/usr/bin/psi.debug #remove static debug file
+	rm $(CURDIR)/debian/psi-plus-webkit/usr/bin/psi-plus
+	rm $(CURDIR)/debian/psi-plus-webkit/usr/share/applications/psi-plus.desktop
+	cp $(CURDIR)/src/psi-plus $(CURDIR)/debian/psi-plus-webkit/usr/bin/psi-plus-webkit
+	cp $(CURDIR)/debian/psi-plus-webkit.desktop $(CURDIR)/debian/psi-plus-webkit/usr/share/applications/
+	rm -f $(CURDIR)/debian/psi-plus-webkit/usr/bin/psi-plus.debug #remove static debug file
 	#Deleting temporary files
-	rm -f $(CURDIR)/debian/psi-plus/usr/share/psi/COPYING
-	rm -f $(CURDIR)/debian/psi-plus/usr/share/psi/README
-	rm -f $(CURDIR)/debian/psi-plus-webkit/usr/share/psi/COPYING
-	rm -f $(CURDIR)/debian/psi-plus-webkit/usr/share/psi/README
+	rm -f $(CURDIR)/debian/psi-plus/usr/share/psi-plus/COPYING
+	rm -f $(CURDIR)/debian/psi-plus/usr/share/psi-plus/README
+	rm -f $(CURDIR)/debian/psi-plus-webkit/usr/share/psi-plus/COPYING
+	rm -f $(CURDIR)/debian/psi-plus-webkit/usr/share/psi-plus/README
 
 override_dh_install:
 	dh_install
@@ -60,7 +60,7 @@
 	[ ! -f Makefile ] || $(MAKE) distclean
 	rm -f config.sub config.guess configure
 
-	rm -f psi src/psi src/config.h
+	rm -f psi-plus src/psi-plus src/config.h
 	rm -f Makefile src/Makefile libpsi/psiwidgets/Makefile
 	rm -f conf.pri extra.pri conf.log
 	rm -f src/.qmake.internal.cache
