diff --git a/Debian/eiskaltdcpp/debian/control b/Debian/eiskaltdcpp/debian/control
index 474b01e..57edcf2 100644
--- a/Debian/eiskaltdcpp/debian/control
+++ b/Debian/eiskaltdcpp/debian/control
@@ -2,7 +2,7 @@ Source: eiskaltdcpp
 Section: net
 Priority: optional
 Maintainer: Boris Pek <tehnick-8@mail.ru>
-Build-Depends: debhelper (>= 7), cmake (>= 2.6.0), autotools-dev, fakeroot,
+Build-Depends: debhelper (>= 7), cmake (>= 2.6.0), po-debconf, autotools-dev, fakeroot,
  libqt4-dev (>=4.4.0), qt4-dev-tools (>=4.4.0), qt4-qtconfig (>=4.4.0),
  libgtk2.0-dev, libglade2-dev, libnotify-dev, liblua5.1-0-dev, libpcre3-dev,
  libboost-dev | libboost1.42-dev | libboost1.40-dev | libboost1.39-dev | libboost1.38-dev | libboost1.37-dev | libboost1.35-dev,
@@ -15,9 +15,11 @@ Architecture: all
 Depends: ${misc:Depends},
  eiskaltdcpp-qt (>= ${binary:Version}),
  eiskaltdcpp-gtk (>= ${binary:Version}),
+ eiskaltdcpp-daemon (>= ${binary:Version}),
  eiskaltdcpp-emoticons,
  eiskaltdcpp-scripts,
  eiskaltdcpp-sounds
+Suggests: eiskaltdcpp-geoip
 Replaces: eiskaltdcpp-unstable
 Conflicts: eiskaltdcpp-unstable
 Description: EiskaltDC++ meta-package
@@ -85,11 +87,33 @@ Description: sound files for EiskaltDC++
  DC from Neomodus, DC++ and derivatives. EiskaltDC++ also interoperates with
  all common DC hub software.
 
-Package: libeiskaltdcpp2.1.x
+Package: eiskaltdcpp-geoip
+Architecture: all
+Depends: ${misc:Depends},
+ unzip, curl,
+ eiskaltdcpp-qt | eiskaltdcpp-gtk
+Replaces: eiskaltdcpp-geoip-unstable
+Conflicts: eiskaltdcpp-geoip-unstable
+Description: Geo IP Database for EiskaltDC++
+ This package contains script for updating the Geo IP database for EiskaltDC++.
+ .
+ Database in text format (GeoIPCountryWhois.csv) will be downloaded from
+ official site (http://maxmind.com/) if it's required.
+ .
+ EiskaltDC++ is a cross-platform program that uses the Direct Connect
+ and ADC protocol. It is compatible with other DC clients, such as the original
+ DC from Neomodus, DC++ and derivatives. EiskaltDC++ also interoperates with
+ all common DC hub software.
+
+Package: libeiskaltdcpp2.2
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Replaces: libdcpp2.0.x, libdcpp2.1.x, libeiskaltdcpp2.1.x-unstable
-Conflicts: libdcpp2.0.x, libdcpp2.1.x, libeiskaltdcpp2.1.x-unstable
+Replaces: libdcpp2.0.x, libdcpp2.1.x,
+ libeiskaltdcpp2.1.x-unstable, libeiskaltdcpp2.2-unstable,
+ libeiskaltdcpp2.1.x,
+Conflicts: libdcpp2.0.x, libdcpp2.1.x,
+ libeiskaltdcpp2.1.x-unstable, libeiskaltdcpp2.2-unstable,
+ libeiskaltdcpp2.1.x,
 Description: DC++ kernel library from EiskaltDC++
  This package installs only DC++ kernel library which required EiskaltDC++.
  .
@@ -98,14 +122,16 @@ Description: DC++ kernel library from EiskaltDC++
  DC from Neomodus, DC++ and derivatives. EiskaltDC++ also interoperates with
  all common DC hub software.
 
-Package: libeiskaltdcpp2.1.x-dbg
+Package: libeiskaltdcpp2.2-dbg
 Section: debug
 Priority: extra
 Architecture: any
 Depends: ${misc:Depends},
- libeiskaltdcpp2.1.x (= ${binary:Version})
-Replaces: libeiskaltdcpp2.1.x-unstable-dbg
-Conflicts: libeiskaltdcpp2.1.x-unstable-dbg
+ libeiskaltdcpp2.2 (= ${binary:Version})
+Replaces: libeiskaltdcpp2.1.x-dbg,
+ libeiskaltdcpp2.1.x-unstable-dbg, libeiskaltdcpp2.2-unstable-dbg
+Conflicts: libeiskaltdcpp2.1.x-dbg,
+ libeiskaltdcpp2.1.x-unstable-dbg, libeiskaltdcpp2.2-unstable-dbg
 Description: DC++ kernel from EiskaltDC++ (debug symbols)
  This package contains the debugging symbols for EiskaltDC++ kernel library.
  .
@@ -117,7 +143,7 @@ Description: DC++ kernel from EiskaltDC++ (debug symbols)
 Package: eiskaltdcpp-qt
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends},
- libeiskaltdcpp2.1.x (= ${binary:Version}),
+ libeiskaltdcpp2.2 (= ${binary:Version}),
  eiskaltdcpp-qt-data (= ${source:Version}),
  eiskaltdcpp-common
 Recommends: libqtscript4-core, libqtscript4-gui, libqtscript4-network, libqtscript4-xml
@@ -141,7 +167,8 @@ Breaks: eiskaltdcpp (<< 2.1.0)
 Suggests: php5-cli,
  eiskaltdcpp-emoticons,
  eiskaltdcpp-scripts,
- eiskaltdcpp-sounds
+ eiskaltdcpp-sounds,
+ eiskaltdcpp-geoip
 Description: data files for EiskaltDC++ Qt
  This package contains the data files for EiskaltDC++ Qt.
  .
@@ -156,7 +183,7 @@ Priority: extra
 Architecture: any
 Depends: ${misc:Depends},
  eiskaltdcpp-qt (= ${binary:Version}),
- libeiskaltdcpp2.1.x-dbg (= ${binary:Version})
+ libeiskaltdcpp2.2-dbg (= ${binary:Version})
 Replaces: eiskaltdcpp-qt-unstable-dbg
 Conflicts: eiskaltdcpp-qt-unstable-dbg
 Description: EiskaltDC++ GUI on Qt (debug symbols)
@@ -170,7 +197,7 @@ Description: EiskaltDC++ GUI on Qt (debug symbols)
 Package: eiskaltdcpp-gtk
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends},
- libeiskaltdcpp2.1.x (= ${binary:Version}),
+ libeiskaltdcpp2.2 (= ${binary:Version}),
  eiskaltdcpp-gtk-data (= ${source:Version}),
  eiskaltdcpp-common
 Replaces: eiskaltdcpp-gtk-unstable
@@ -192,7 +219,8 @@ Conflicts: eiskaltdcpp-gtk-data-unstable
 Suggests: php5-cli,
  eiskaltdcpp-emoticons,
  eiskaltdcpp-scripts,
- eiskaltdcpp-sounds
+ eiskaltdcpp-sounds,
+ eiskaltdcpp-geoip
 Description: data files for EiskaltDC++ Gtk
  This package contains the data files for EiskaltDC++ Gtk.
  .
@@ -207,7 +235,7 @@ Priority: extra
 Architecture: any
 Depends: ${misc:Depends},
  eiskaltdcpp-gtk (= ${binary:Version}),
- libeiskaltdcpp2.1.x-dbg (= ${binary:Version})
+ libeiskaltdcpp2.2-dbg (= ${binary:Version})
 Replaces: eiskaltdcpp-gtk-unstable-dbg
 Conflicts: eiskaltdcpp-gtk-unstable-dbg
 Description: EiskaltDC++ GUI on Gtk (debug symbols)
@@ -217,3 +245,34 @@ Description: EiskaltDC++ GUI on Gtk (debug symbols)
  and ADC protocol. It is compatible with other DC clients, such as the original
  DC from Neomodus, DC++ and derivatives. EiskaltDC++ also interoperates with
  all common DC hub software.
+
+Package: eiskaltdcpp-daemon
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends},
+ libeiskaltdcpp2.2 (= ${binary:Version})
+Replaces: eiskaltdcpp-daemon-unstable
+Conflicts: eiskaltdcpp-daemon-unstable
+Description: EiskaltDC++ daemon
+ This package installs only EiskaltDC++ daemon (without any GUI).
+ .
+ EiskaltDC++ is a cross-platform program that uses the Direct Connect
+ and ADC protocol. It is compatible with other DC clients, such as the original
+ DC from Neomodus, DC++ and derivatives. EiskaltDC++ also interoperates with
+ all common DC hub software.
+
+Package: eiskaltdcpp-daemon-dbg
+Section: debug
+Priority: extra
+Architecture: any
+Depends: ${misc:Depends},
+ eiskaltdcpp-daemon (= ${binary:Version}),
+ libeiskaltdcpp2.2-dbg (= ${binary:Version})
+Replaces: eiskaltdcpp-daemon-unstable-dbg
+Conflicts: eiskaltdcpp-daemon-unstable-dbg
+Description: EiskaltDC++ daemon (debug symbols)
+ This package contains the debugging symbols for EiskaltDC++ daemon.
+ .
+ EiskaltDC++ is a cross-platform program that uses the Direct Connect
+ and ADC protocol. It is compatible with other DC clients, such as the original
+ DC from Neomodus, DC++ and derivatives. EiskaltDC++ also interoperates with
+ all common DC hub software.
diff --git a/Debian/eiskaltdcpp/debian/copyright b/Debian/eiskaltdcpp/debian/copyright
index 408cb48..f83217c 100644
--- a/Debian/eiskaltdcpp/debian/copyright
+++ b/Debian/eiskaltdcpp/debian/copyright
@@ -88,7 +88,7 @@ Files:
  extra/freespace.cpp
 Copyright:
  Copyright 2001-2010 Jacek Sieka <arnetheduck@gmail.com>
- Copyright 2008-2009 Big Muscle (http://strongdc.sf.net)
+ Copyright 2008-2010 Big Muscle (http://strongdc.sf.net)
  Copyright 1996-2002 Florian Schintke
  Copyright 1997 Makoto Matsumoto and Takuji Nishimura <matumoto@math.keio.ac.jp>
  Copyright 2010 cologic <ne5@parsoma.net>
@@ -156,8 +156,8 @@ Files:
  eiskaltdcpp-qt/icons/*
  eiskaltdcpp-gtk/icons/*
 Copyright:
- Copyright 2010 Oxygen icons theme team (see http://www.oxygen-icons.org/)
  Copyright 2010 Uladzimir Bely (WiseLord) <wiselord1983@gmail.com>
+ Copyright 2010 Oxygen icons theme team (see http://www.oxygen-icons.org/)
 License:
  LGPL-3
  .
@@ -165,6 +165,16 @@ License:
  Public License can be found in `/usr/share/common-licenses/LGPL-3'.
 
 Files:
+ eiskaltdcpp-qt/icons/appl/faenza/*
+Copyright:
+ Copyright 2010-2011 Matthieu James (Tiheum) <matthieu.james@gmail.com>
+License:
+ GPL-3
+ .
+ On Debian systems, the complete text of the GNU General
+ Public License can be found in `/usr/share/common-licenses/GPL-3'.
+
+Files:
  sounds/*.wav
  emoticons/*
 Copyright:
@@ -194,6 +204,7 @@ License:
 Files:
  eiskaltdcpp-qt/src/FlowLayout.cpp
  eiskaltdcpp-qt/src/FlowLayout.h
+ eiskaltdcpp-qt/src/qtsingleapp/*
  eiskaltdcpp-qt/qtscripts/tic-tac-toe/*
 Copyright:
  Copyright 2010 Nokia Corporation and/or its subsidiary(-ies).
@@ -237,6 +248,19 @@ Copyright:
  Copyright 2009-2010 freedcpp (http://code.google.com/p/freedcpp)
  Copyright 2009-2010 Leliksan Floyd <leliksan@Quadrafon2>
  Copyright 2003 Bastien Nocera <hadess@hadess.net>
+ Copyright 2010 Mank <mank@besthub.eu>
+ Copyright 2010 Eugene Petrov <dhamp@ya.ru>
+License:
+ GPL-2 or later
+ .
+ On Debian systems, the complete text of the GNU General
+ Public License can be found in `/usr/share/common-licenses/GPL-2'.
+
+Files:
+ nasdc/*
+Copyright:
+ Copyright 2009-2010 Alexandr Tkachev <tka4ev@gmail.com>
+ Copyright 2010 Eugene Petrov <dhamp@ya.ru>
 License:
  GPL-2 or later
  .
diff --git a/Debian/eiskaltdcpp/debian/rules b/Debian/eiskaltdcpp/debian/rules
index 1cf6aef..025f36e 100755
--- a/Debian/eiskaltdcpp/debian/rules
+++ b/Debian/eiskaltdcpp/debian/rules
@@ -13,14 +13,17 @@ BUILDDIR = builddir
 CMAKEOPTS = -DCMAKE_BUILD_TYPE=RelWithDebInfo \
             -DCMAKE_INSTALL_PREFIX=/usr \
             -DUSE_MINIUPNP=OFF -DLOCAL_MINIUPNP=OFF \
-            -DLUA_SCRIPT=ON -DWITH_LUASCRIPTS=ON \
+            -DLUA_SCRIPT=OFF -DWITH_LUASCRIPTS=OFF \
             -DPERL_REGEX=ON \
             -DWITH_SOUNDS=ON \
             -DUSE_QT=ON \
             -DUSE_ASPELL=ON -DFREE_SPACE_BAR_C=ON -DUSE_JS=ON \
+            -DUSE_QT_SQLITE=OFF \
             -DUSE_GTK=ON \
             -DUSE_LIBGNOME2=OFF -DUSE_LIBNOTIFY=ON \
             -DCREATE_MO=ON -DUPDATE_PO=OFF \
+            -DNO_UI_DAEMON=ON \
+            -DXMLRPC_DAEMON=OFF \
             ..
 
 # This option is for daily builds of the EiskaltDC++ packages on Launchpad
@@ -52,6 +55,9 @@ install: build
 	dh_prep
 	dh_installdirs
 	cd $(BUILDDIR) && $(MAKE) install DESTDIR=$(CURDIR)/debian/tmp
+	mkdir -p $(CURDIR)/debian/$(eiskaltdcpp-geoip)/usr/share/eiskaltdcpp/
+	cp -f $(CURDIR)/debian/update_geoip $(CURDIR)/debian/$(eiskaltdcpp-geoip)/usr/share/eiskaltdcpp/
+	chmod uog=rwx $(CURDIR)/debian/$(eiskaltdcpp-geoip)/usr/share/eiskaltdcpp/update_geoip
 binary-indep: build install
 	dh_testdir -i
 	dh_testroot -i
@@ -72,15 +78,16 @@ binary-arch: build install
 	dh_installdocs -a
 	dh_installman -a
 	dh_link -a
-	dh_strip -Nlibeiskaltdcpp2.1.x -Neiskaltdcpp-qt -Neiskaltdcpp-gtk
-	dh_strip -plibeiskaltdcpp2.1.x --dbg-package=libeiskaltdcpp2.1.x-dbg
+	dh_strip -Nlibeiskaltdcpp2.2 -Neiskaltdcpp-qt -Neiskaltdcpp-gtk -Neiskaltdcpp-daemon
+	dh_strip -plibeiskaltdcpp2.2 --dbg-package=libeiskaltdcpp2.2-dbg
 	dh_strip -peiskaltdcpp-qt --dbg-package=eiskaltdcpp-qt-dbg
 	dh_strip -peiskaltdcpp-gtk --dbg-package=eiskaltdcpp-gtk-dbg
+	dh_strip -peiskaltdcpp-daemon --dbg-package=eiskaltdcpp-daemon-dbg
 	dh_compress -a
 	dh_fixperms -a
-	dh_makeshlibs -plibeiskaltdcpp2.1.x
+	dh_makeshlibs -plibeiskaltdcpp2.2
 	dh_installdeb -a
-	dh_shlibdeps -a -ldebian/libeiskaltdcpp2.1.x/usr/lib
+	dh_shlibdeps -a -ldebian/libeiskaltdcpp2.2/usr/lib
 	dh_installdebconf -a
 	dh_gencontrol -a
 	dh_md5sums -a
